avocado-setup.py

```diff
@@ -266,6 +267,18 @@ def install_optional_plugin(plugin):
         # plugin already installed
         pass
 
+def configure_prepostscript(repos, scripttype):
+    i=0
+    cmd= 'mkdir -p /etc/avocado/scripts/job/%s.d' % scripttype
+    helper.runcmd(cmd, ignore_status=True, info_str="%s script directory created" % scripttype)
+    for repo in repos:
+        i=i+1
+        path= "/etc/avocado/scripts/job/%s.d/%sjobscript_repo%d.sh" %(scripttype,scripttype , i)
+        cmd="wget -O %s  %s;chmod 777 %s;%s" % (path, repo, path, path)
```

Comment:
you meant by `cp` cmd?, path is not from http or ftp, why `wget` ?

Reply:
here again same confusion

Reply:
as i expect this always hosted in the remote server so did that way